node/bundler.ts

```diff
@@ -173,18 +172,31 @@ const getBasePath = (sourceDirectories: string[], inputBasePath?: string) => {
   return commonPathPrefix(sourceDirectories)
 }
 
-export const addGeneratorFieldIfMissing = (
-  declaration: Declaration,
-  functions: EdgeFunction[],
-  internalFunctionsPath?: string,
-) => {
-  const fullFuncPath = functions?.find((func) => func.name === declaration.function)?.path
-
+const addMissingConfigFields = ({
+  name,
+  config,
+  functions,
+  declarations,
+  internalFunctionsPath,
+}: {
```

Comment:
nit: Could we move these options into an interface?

Reply:
I shall!